dhis-2/dhis-web-api/src/main/resources/openapi/TrackedEntitiesExportController.md

```diff
@@ -166,6 +185,18 @@ Get tracked entities with given UID(s).
 Get tracked entities with events assigned to users according to the specified user mode. By default,
 all events will be retrieved, regardless of whether a user is assigned.
 
+- When `assignedUserMode=ALL` or no `assingedUserMode`(default) is given, tracked entities and
+  enrollments are returned irrespective of wether they contain events assigned to a user or not.
+  This is the default org unit mode. 
+- When `assignedUserMode=CURRENT` tracked entities and enrollments that has at least one event
+  assigned to the logged in user will be returned.
+- When `assignedUserMode=ANY` tracked entities and enrollments that has at least one event with
+  an assigned user will be returned.
+- When `assignedUserMode=NONE` tracked entities with no events assigned to any user will be
```

Comment:
before you mention TE and EN and here only TEs, why is that?

Reply:
I removed the mention of EN to avoid any confusion